airflow/models/taskinstance.py

```diff
@@ -1634,6 +1634,7 @@ def _handle_reschedule(
     # see https://github.com/apache/airflow/pull/21362 for more info
     session.add(
         TaskReschedule(
+            ti.ti_id,
             ti.task_id,
             ti.dag_id,
             ti.run_id,
```

Comment:
We should think if we need the old PK columns or if just id is enough.

(I'm not sure either way but we should think about it)

Reply:
Personally I’d prefer the extra fields to go away. They can stay as associated proxies, but there’s close to zero benefit keeping unnormalised columns in those models.

Reply:
The new `ti_id` makes old PK columns to be redundant.